File: Data Structures & Algorithms/majority-element/submission-0.java
class Solution {
    public int majorityElement(int[] nums) {
        int count =0;
        int variable =0;

        for(int i=0;i<nums.length;i++){
            if(count==0){
                count =1;
                variable = nums[i];
            }
            else if(nums[i] == variable){
                count++;
            }
            else {
                count--;
            }
        }
        int count1=0;

        for(int i=0;i<nums.length;i++){
            if(variable==nums[i]){
                count1++;
            }

        }

        if(count1>nums.length/2){
            return variable;
        }

        return -1;
        
    }
}

File: Data Structures & Algorithms/remove-element/submission-0.py
class Solution:
    def removeElement(self, nums: List[int], val: int) -> int:          
      i,j=0,0
      while j<len(nums):
            ele=nums[j]
            if ele!=val:
                  nums[i],nums[j]=nums[j] , nums[i]
                  i+=1
                  j+=1
            
            else:
                  j+=1
            
                
            
      return i



        